### ISSUE TITLE
NameError: name 'long' is not defined When Using Python 3

### ISSUE THREAD

**Issue by napperley** [ecometrica/grandfatherson#3]
When using Python 3 instead of Python 2.7 the following error appears when invoking the **grandfatherson.dates_to_delete** function:

```
File "/usr/local/lib/python3.4/dist-packages/grandfatherson/__init__.py", line 220, in dates_to_delete
    firstweekday=firstweekday, now=now)
  File "/usr/local/lib/python3.4/dist-packages/grandfatherson/__init__.py", line 204, in dates_to_keep
    firstweekday=firstweekday, now=now)
  File "/usr/local/lib/python3.4/dist-packages/grandfatherson/__init__.py", line 172, in to_keep
    filters.Minutes.filter(datetimes, number=minutes, now=now) |
  File "/usr/local/lib/python3.4/dist-packages/grandfatherson/filters.py", line 55, in filter
    if number < 0 or not isinstance(number, (int, long)):
NameError: name 'long' is not defined
```

The current [recommendation](http://stackoverflow.com/questions/14904814/nameerror-global-name-long-is-not-defined) when migrating from Python 2.7 to Python 3 is to use an **int** instead of a **long**.

**Comment by EugeneDae** [comment]
bump

**Comment by rory-geoghegan-ecometrica** [comment]
In python 2 (on a 64bit platform), `isinstance(int(2**66), int) == False`. However, since the minimum filter is seconds, I think we can limit people to that bound in python 2.7.

**Comment by rory-geoghegan-ecometrica** [comment]
Should be fixed in version 1.3: https://pypi.python.org/pypi/GrandFatherSon/1.3

**Comment by EugeneDae** [comment]
fuk yes 👍  I can now use your awesome package directly via pip
thank you
